libobs/obs-windows.c

```diff
@@ -64,13 +64,42 @@ char *find_libobs_data_file(const char *file)
 	struct dstr path;
 	dstr_init(&path);
 
+	WCHAR *env_path = 0;
+	DWORD env_path_sz = 
+		GetEnvironmentVariableW(L"OBS_DATA_PATH", NULL, 0);
+
+	env_path = bmalloc(env_path_sz * sizeof(WCHAR));
+
+	if (env_path_sz) {
+		GetEnvironmentVariableW(
+		    L"OBS_DATA_PATH",
+		    env_path, env_path_sz);
+	}
+
+	if (env_path) {
+		char *env_path_utf8 = 0;
```

Comment:
Minor/insignificant point here, but this should probably be "= NULL;" rather than "= 0;".  They're the same, but NULL is more proper to use for null pointer values.  A general "avoid magic numbers" rule sort of thing.